### ISSUE TITLE
Provide an option to skip ssl for connections to the artifactory server

### ISSUE THREAD

**Issue by gberche-orange** [spring-io/artifactory-resource#65]
* As an artifactory-resource user
* In order to upload artefacts to an artifactory server on the intranet serving TLS over trusted custom rootCAs
* I need artifactory-resource to either support "skipSsl" to bypass certificate checks, or that default system CA certs [automatically mounted by concourse](https://concourse-ci.org/implementing-resource-types.html#resource-certs) into the resource container at `/etc/ssl/certs` to be transparently used.

Currently, a curl command within the resource container to the artifactory server works, but the artifactory-resource fails with the following redacted trace

```
un.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target; nested exception is javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```
Full stack trace at https://gist.github.com/gberche-orange/f78abca01d1d30ba91a7132361327ec8

I did not find a way to change the default behavior without modifying the java source code and rebuilding the docker image.

**Comment by mbhave** [comment]
Adding a skipssl options sounds good to me. I also wonder if we could automatically configure the truststore if the `/etc/ssl/certs` file exists.

**Comment by gberche-orange** [comment]
thanks @mbhave!

Automatically mounting configure the truststore if the /etc/ssl/certs file exists would be a useful addition too (in a similar way to cloudfoundry java buildpack, see https://github.com/cloudfoundry/java-buildpack/blob/03e8eec73618ca95c640fc5e834815491555f025/docs/framework-container_security_provider.md)

**Comment by mbhave** [comment]
I've created a [new issue](#72) for that. We can keep this one for adding the option to skip ssl.

**Comment by philwebb** [comment]
We'd rather not give the option to disable HTTPS if at all possible.

@gberche-orange I've pushed a fix for #72 that _should_ mean we pickup concourse certs. Can you give 0.0.13-SNAPSHOT a try to see if it works for you. If not, we can reopen this one.

**Comment by gberche-orange** [comment]
thanks @philwebb !
